files/settings-default.cfg

```diff
@@ -174,6 +174,13 @@ buffer cache min = 14
 # to this much memory until old buffers get purged.
 buffer cache max = 16
 
+# Specifies whether to enable HRTF processing. Valid values are: -1 = auto,
+# 0 = off, 1 = on.
+hrtf enable = -1
```

Comment:
Out of curiosity, how well does 'auto' work? You probably don't want to use HRTF for speakers, so how does OpenAL know if a device is headphone or speaker?